geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/data/KafkaDataStoreFactory.scala

```diff
@@ -112,7 +114,17 @@ object KafkaDataStoreFactory extends GeoMesaDataStoreInfo with LazyLogging {
       KafkaDataStore.ProducerConfig(props)
     }
 
-    val serialization = org.locationtech.geomesa.features.SerializationType.withName(SerializationType.lookup(params))
+    val schemaRegistryUrl = SchemaRegistryUrl.lookupOpt(params)
+
+    val serialization = {
+      val ser = org.locationtech.geomesa.features.SerializationType.withName(SerializationType.lookup(params))
```

Comment:
We could tweak this a little so that it doesn't use the default value (`kryo`) if nothing is specified. Like:
```
val ser = if (schemaRegistryUrl.isDefined && !params.containsKey(SerializationType.key)) {
  org.locationtech.geomesa.features.SerializationType.CONFLUENT
} else {
  org.locationtech.geomesa.features.SerializationType.withName(SerializationType.lookup(params))
}
```
(doing this from memory so code may not be exactly right)